### ISSUE TITLE
mail.com not avaible on SailfishOS

### ISSUE THREAD

**Issue by falematte** [deltachat/deltachat-android#271]
After the installation everything worked fine and I succesfully send 3 messages to other emails from my mail.com account. Now messages are not send and an error is displayed saying that the mailbox is not avaible.

**Delta Chat version**
0.15.0

**Comment by Hocceruser** [comment]
Have you tried to shut down your device and put it on again? If yes, could you try to uninstall and re-install Delta Chat?

**Comment by falematte** [comment]
After re-installaing messages are sended. After some time I got the same error... It appears to give this error randomly. Sometimes during the day messages are sent... Do you think this is related with "mail.com"? Would you suggest other email services which work good with delta chat?

**Comment by Hocceruser** [comment]
> Would you suggest other email services which work good with delta chat?

Well, I didn't have any problems with GMX but you have to activate IMAP first: 
https://support.gmx.com/pop-imap/toggle.html or 
https://img.ui-portal.de/hilfe/helpcenter/de/gmxnet/videos/EMail_ActivatePOP3-IMAP_gmxnet.mp4 (video).

I don't know if this will solve your problem though, but I'd say that it's worth a try...

Another (better?) approach would be to read the logs to find (and fix) the bug in Delta Chat but I don't know how to do this (I think that you have to use ADB).

**Comment by r10s** [comment]
if the issue is still valid for the most recent version, feel free to reopen or to add a post to the user forum at https://support.delta.chat/

**Comment by KingDuckZ** [comment]
Even though this is closed I don't see any fix mentioned. I'm having the same issue on v0.20.0 on SailfishOS. I'm also using a gmx account.
It's the exact same situation, it used to work and now it stopped working, though I still receive other people's messages.

Maybe not related but incoming messages also don't get deleted from my mailbox after reading them. It started with someone being added to the chat group, from that invitation message on messages are not deleted anymore, but I can see all the new messages from the app.

**Comment by r10s** [comment]
@falematte thank you for coming back to this issue.

well in general, to fix something, information to reproduce the problem are needed.
if no one can reproduce the problem and no additional information to issues are added for a longer time, it is usual that issues are closed - esp. if there are only few information in the issue and/or no one can confirm it. computers are complex systems and problems may have completely different reasons. 

also closing issues does not mean they are deleted, you may also regard it as "not solvable today" or "postponed" :)

eg. the information about SailfishOS is new, but additional information as logs, if other accounts do work, will still be required to so sth. on this issue. also a developer (with time) using SailfishOS is needed. try to add as many information as you can.

wrt. providers that are known to work, see https://support.delta.chat/t/provider-overview/56/2?u=compl4xx

**Comment by KingDuckZ** [comment]
I didn't mean to hijack the bug report, I was just reporting that I'm also affected at my current version.
By the way I'm not falematte.

I have no idea how to post logs here, if it's not too complicated I can give it a shot.

**Comment by r10s** [comment]
eg. you can post logs directly in your reply or add and long to a [gist](http://gist.github.com/)

**Comment by KingDuckZ** [comment]
Yes, but what do you want me to post? Like, what logs? How do I retrieve them? When I said "if it's easy" that's the part I was talking about.

**Comment by r10s** [comment]
ahh, i see :) from the template when filing a new issue:
```
If applicable, debug logs can be copied from within the Delta Chat app: 

Open the _Settings_ menu -> _About_ -> _version number|Info_.

Alternatively from the Android system log:

` adb logcat -v time -s DeltaChat `

This log contains private data (e.g. mail address, provider information) which shall be removed or anonymised prior to posting.
```

**Comment by KingDuckZ** [comment]
Ok, I've just commented to an existing issue, so I had never seen that text! Either way, here it is (hopefully I've removed all the email addresses and names): https://alarmpi.no-ip.org/kamokan/ck?colourless

**Comment by r10s** [comment]
thanks.

the main issue seems to be that one of the used recipient cannot be added to gmx, see log lines as `SMTP failed to add recipient`, in the code this is here: https://github.com/deltachat/deltachat-core/blob/master/src/dc_smtp.c#L298

this may not be an error of Delta Chat but just a non-working address - or even an issue with the provider.

wondering, what happens if you create a new chat and use eg. `a@deltachat.de` (one of our testing addresses) as recipient.

**Comment by r10s** [comment]
the gmx error is `invalid DNS MX or A/AAAA resource record`

and here are some information about this issue:
https://postmaster.gmx.com/en/error-messages#invaliddnsmxora

so, might be there is sth. wrong with the sending server. i am no mail expert, so, i cannot help further here.

**Comment by angelo-fuchs** [comment]
@KingDuckZ Is the problem still present in a current (0.100+) version? (Should be on f-droid in the next days, else on our download page)

**Comment by r10s** [comment]
moving this to core ...
